tests/integration/security/testdata/authz/v1beta1-operation.yaml.tmpl

```diff
@@ -0,0 +1,49 @@
+---
+# The following policy will allow every request other then GET request
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: policy-{{ .b }}-allow
+  namespace: "{{ .Namespace }}"
+spec:
+  selector:
+    matchLabels:
+      "app": "{{ .b }}"
+  action: ALLOW
+  rules:
+  - to:
+    - operation:
+        notMethods : ["GET"]
+---
+# The following policy won't allow request from hosts "deny.com"
```

Comment:
hosts is the destination, so it's not `from hosts "deny.com"` but `to host deny.com`